README.md

```diff
@@ -2195,6 +2195,15 @@ $ cd build/test
 $ ctest . -V
 ```
 
+##### Linking to libc++
```

Comment:
This doesn't need to be documented in the README. Users who want to use libc++ will have to look up how that can be done with their own compiler and act accordingly and you can't possibly know all the varieties of compilers the users will use. This just adds noise imo.

If I want to use libc++ I will do it my own way, depending on what I'm doing at the time.

Reply:
Users will find information how to link to libc++ but it will be much harder to figure out why the library does not build on older libstdc++ versions and how to make it work. I think it would be helpful for a significant number of users if this is explained in the README.

If you prefer to not have explicit instructions how to make it work with older Clang versions, I can remove it. But I would keep at least a note that says that it will not work out of the box for these versions.

Reply:
Well you bring up a valid point that older libraries will linger around for a while, but personally I would just prefer a notice, instead of explicit instructions.